### ISSUE TITLE
Failure messages include unwanted Some

### ISSUE THREAD

**Issue by mcovarr** [broadinstitute/cromwell#1893]
Failure metadata will look something like:

```
  "failures": [{
    "failure": "Task 874ad75a-e19e-4b53-8225-746df1436c51:ValidateAggregatedSamFile was preempted for the 1st time. The call will be restarted with another preemptible VM (max preemptible attempts number is 3).\nError code 10. Message: Some(14: VM ggp-15150083938845849899 stopped unexpectedly.)",
    "timestamp": "2016-12-18T05:54:07.296Z"
  }],
```

**Comment by geoffjentry** [comment]
If only we ADT-ized case classes with a single `Option`al member ...

**Comment by geoffjentry** [comment]
Closed by #1899
